specification/indices/put_settings/IndicesPutSettingsRequest.ts

```diff
@@ -40,6 +40,7 @@ export interface Request extends RequestBase {
     preserve_existing?: boolean
     timeout?: Time
   }
+  /** @identifier settings */
```

Comment:
As discussed, this should be `body: IndexSettings` and we should remove `IndexSettingsBody`. The nesting in a `settings` property is a legacy representation that ES server eliminates if present before processing the request further.